### ISSUE TITLE
Error after windows install from setup.exe

### ISSUE THREAD

**Issue by AfterSpencer** [hubblestack/hubble#513]
Installed from https://github.com/hubblestack/hubble/releases/download/v2.4.7/Hubble-v2.4.7-Setup.exe on Windows 10 for testing and came across this error:

```
C:\Program Files (x86)\Hubble>hubble.exe
An Error occurred while loading the config: object of type 'NoneType' has no len()
Traceback (most recent call last):
  File "hubble.py", line 5, in <module>
  File "hubblestack\daemon.py", line 49, in run
    load_config()
  File "hubblestack\daemon.py", line 468, in load_config
    __opts__ = salt.config.minion_config(parsed_args.get('configfile'))
  File "site-packages\salt\config\__init__.py", line 2422, in minion_config
  File "site-packages\salt\config\__init__.py", line 3783, in apply_minion_config
TypeError: object of type 'NoneType' has no len()
Failed to execute script hubble
```

**Comment by basepi** [comment]
That's a new one. I'll get a ticket into our backlog to look into this.

**Comment by basepi** [comment]
https://jira.corp.adobe.com/browse/CSTREQ-45943

**Comment by AfterSpencer** [comment]
@alias454 commenting those out allows hubble.exe --version to run.

Thanks.

**Comment by basepi** [comment]
Fixed in #553
